README.md

```diff
@@ -19,13 +19,60 @@ To cope with it, Zed's flatpak defaults can be changed to:
     $ flatpak override --user dev.zed.Zed --env=ZED_FLATPAK_NO_ESCAPE=1
   ```
 
-### Execute commands on the host system
+### Execute commands in the host system
```

Comment:
Strictly speaking, binaries, scripts, ... are executed on a runtime env (OS, interpreter, ...), not in.